dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/registry/RegistryClient.java

```diff
@@ -144,7 +145,9 @@ public Map<String, String> getServerMaps(NodeType nodeType) {
             String path = rootNodePath(nodeType);
             Collection<String> serverList = getServerNodes(nodeType);
             for (String server : serverList) {
-                serverMap.putIfAbsent(server, get(path + SINGLE_SLASH + server));
+                if (NetUtils.isLegalAddress(server)) {
+                    serverMap.putIfAbsent(server, get(path + SINGLE_SLASH + server));
+                }
```

Comment:
I think the check should throw exception, rather return false, there's must be some unexpected errors if it doesn't pass the check.

Reply:
> I think the check should throw exception, rather return false, there's must be some unexpected errors if it doesn't pass the check.

may no any errors. because old DS version will contain `default` node, and no any update steps will remove it. may do not need throw a exception to print log each 10s. WDYT.

Reply:
> may no any errors. because old DS version will contain `default` node, and no any update steps will remove it. may do not need throw a exception to print log each 10s. WDYT.

Should we consider other failed cases except for default node? I just wonder how do users realize the failed check if there's an invalid address saved in register center.